funderfinder/sources/github_sponsors.py

```diff
@@ -51,7 +53,7 @@ def get_gh_org_funding_json(self, org: str) -> Any:
         data = result.json()
         return data
 
-    def get_org_funder_count(self, org: str) -> dict:
+    def get_org_funder_count(self, org: str) -> int:
         """
         Retrieves GitHub sponsors statistics for a GitHub organization.
         :param org: identifier for the GitHub organization
```

Comment:
Should `:return: Dict of funding stats` now be `:return: number of org funders`?

Reply:
Good point, will be done in next commit!